test/pure-docker/Vagrantfile

```diff
@@ -1,68 +1,60 @@
 # -*- mode: ruby -*-
```

Comment:
I was planning to copy this entire file before to create a new `test/docker-compose/Vagrantfile` test, is that still a sane approach given that this is now more templating logic than anything else? (everything is done in `servers.yaml` now)

Reply:
I would suggest that the Vagrant file remain as is, or install everything we need for both, then any excecution be done via the shell commands in the `servers.yaml`.